### PR TITLE
JBPM-5084: troubleshooting function for workitem

### PR DESCRIPTION
**Pull request by bxf12315**
@mswiderski 
It is update code, include RetryWorkItemCommand.
Thanks

### CODE REVIEW

File: drools-core/src/main/java/org/drools/core/command/runtime/process/ReTryWorkItemCommand.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
+package org.drools.core.command.runtime.process;
```
Comment (mswiderski) [inline drools-core/src/main/java/org/drools/core/command/runtime/process/ReTryWorkItemCommand.java:17]: missing license header

**Comment by mswiderski** [comment]
ok to test

**Comment by mswiderski** [comment]
could be good to align with formatting, besides that looks good

**Comment by bxf12315** [comment]
@mswiderski  
About code formatting , i use https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/tree/master/ide-configuration/eclipse-configuration/code-style,
and  formatted it , but also have some different .

**Comment by mswiderski** [comment]
jenkins retest this

**Comment by mswiderski** [comment]
jenkins retest this

**Comment by mswiderski** [comment]
+1

**Pull request merged**